src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    static final int COUNT_OF_FIGURES = 5;
+
+    public static void main(String[] args) {
+        FigureSupplier figuresupplier = new FigureSupplier();
```

Comment:
According to the checklist, you should not create a new instance of 'FigureSupplier' in the 'Main' class. Instead, consider moving the logic for generating figures to a separate method within the 'Main' class or to a separate class entirely.